mass2/core/drift_correction.py

```diff
@@ -81,10 +92,13 @@ def learn(
 
 @dataclass
 class DriftCorrection:
+    """A linear correction for gain drift that correlates with pretrigger mean."""
```

Comment:
A linear correction used to attempt remove pretrigger mean correlation with pulse height, maybe be used with other quantities.